README.md

```diff
@@ -11,8 +11,11 @@ Extract the archive or clone the repository.
 
 Graphical Interface: `pycam/run_gui.py`
 
-Scripted Toolpath Processing: `pycam/run_cli.py FLOW_SPECIFICATION_FILE`
+Scripted Toolpath Processing: `pycam/run_cli.py FLOW_SPECIFICATION_FILE.yml`
```

Comment:
In `README.md`, the [FAQ](http://pycam.sourceforge.net/faq.md) link has expired. The new link should be [this](http://pycam.sourceforge.net/faq/)

Reply:
Nice catch! This is also done

Reply:
Great! I think I finished my review. Don't know the next step of your PR.

Reply:
Thanks @jinglingxing, I'll merge this one then!